.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,5 @@
+<!-- Love storybook? Please consider supporting our collective: 👉  https://opencollective.com/storybook/donate -->
```

Comment:
Looks a bit aggressive, doesn't it? What do we want to say to a person creating PR? I would leave it only for issue, and didn't merge until the official publication